src/Settings/Equations/f_re.cpp

```diff
@@ -188,7 +188,7 @@ void SimulationGenerator::ConstructEquation_f_re_kineq(
                             // will in general be different from the quantity n_re
                             for (len_t j = 0; j < np2; j++)
                                 for (len_t i = 0; i < np1; i++)
-                                    finit[offset + j*np1 + i] = fRE->evaluateFullDistribution(ir, xi[j], p[i]);
+                                    finit[offset + j*np1 + i] = fRE->evaluateFullDistributionWithE(ir, xi[j], p[i], E_init[ir]);
```

Comment:
Now the function "evaluateFullDistribution" is never used anywhere, if I'm not mistaken, but that is probably fine